vello/README.md

```diff
@@ -183,7 +187,8 @@ The [`with_winit`](#winit) example supports running on Android, using [cargo apk
 cargo apk run -p with_winit
 ```
 
-> [!TIP]
+> 💡 TIP
+>
```

Comment:
It might be worth removing the web/android/with_winit detailed running instructions from this crate-level readme

I think that should be a post-release follow-up, probably.